### DISCUSSION TITLE
How to install my nuxt app as mobile app

### DISCUSSION THREAD

**Discussion by DouglasMai4** [nuxt/nuxt#16847] [Questions] [answered]
Hi, I would like to be able to install my website made with nuxt as if it were a mobile app, the nuxt docs allow to do this, and it really looks like an app and I would like to do that, but I have no idea how to start, does anyone know show me a way?

**Comment by TheAlexLichter** [comment, marked as answer]
What you want is a PWA. Feel free to take a look at https://github.com/kevinmarrec/nuxt-pwa-module (which is a Nuxt3 version of https://pwa.nuxtjs.org/) as well as https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps in general ☺️
